.gitignore

```diff
@@ -1,6 +1,8 @@
 out
 node_modules
 .vscode-test
+ion-extension-0.2.0.vsix
+test-resources/
```

Comment:
is this a generated directory? What is in `test-resources`?

Reply:
`test-resources` contains the necessary files to setup the 'extest' test suite. It gets generated from 'npm install'.